filters/tracing/tag.go

```diff
@@ -55,5 +63,3 @@ func (f tagFilter) Request(ctx filters.FilterContext) {
 		span.SetTag(f.tagName, v)
```

Comment:
With this chnage this will be executed twice on request and response path.
E.g. `tracingTag("foo", "${request.header.X-Foo}") -> setRequestHeader("X-Foo", "bar")` will set tag to `bar` I think.

Reply:
Thanks for the hint!. I noticed it when reviewing. I'm currently changing it to just add a filter.